xml/System.ComponentModel/VersionConverter.xml

```diff
@@ -178,11 +178,13 @@
         <Parameter Name="value" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="context">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="context">An <see cref="T:System.ComponentModel.ITypeDescriptorContext" /> that provides a format context.</param>
+        <param name="value">The <see cref="T:System.Object" /> to test for validity.</param>
+        <summary>Returns whether the given value object is valid for this type and for the specified context.</summary>
+        <returns>
```

Comment:
nit: remove this endline

Reply:
@carlos what do you mean by removing this endline?

Reply:
GitHub doesn't allow giving multi-line suggestions. I mean merging lines 184 and 185:

```xml
        <returns><see langword="true" /> if the specified value is valid for this object; otherwise, <see langword="false" />.</returns>
```

Reply:
Sure. Addressed this with last commit: 42ad6824eb1173ae18c5d267a155a99dbff2b7d4